doc/cookbook/resilience.md

```diff
@@ -3,361 +3,372 @@
 - [Resilience](#resilience)
   - [Basic: Load Balance](#basic-load-balance)
   - [More Livingness: Resilience of Service](#more-livingness-resilience-of-service)
-    - [CircuitBreaker](#circuitbreaker)
+    - [Circuit Break](#circuit-break)
     - [RateLimiter](#ratelimiter)
-    - [Retryer](#retryer)
-    - [TimeLimiter](#timelimiter)
+    - [Retry](#retry)
+    - [Time Limit](#time-limit)
   - [References](#references)
-    - [CircuitBreaker](#circuitbreaker-1)
+    - [Circuit Break](#circuit-break-1)
     - [RateLimiter](#ratelimiter-1)
-    - [Retryer](#retryer-1)
-    - [TimeLimiter](#timelimiter-1)
+    - [Retry](#retry-1)
     - [Concepts](#concepts)
 
-As a Cloud Native traffic orchestrator, Easegress supports build-in resilience features. It is the ability of your system to react to failure and still remain functional. It's not about avoiding failure, but accepting failure and constructing your cloud-native services to respond to it. You want to return to a fully functioning state quickly as possible.[1]
+As a Cloud Native traffic orchestrator, Easegress supports build-in resilience
+features. It is the ability of your system to react to failure and still
+remain functional. It's not about avoiding failure, but accepting failure
+and constructing your cloud-native services to respond to it. You want to
+return to a fully functioning state quickly as possible.[1]
 
```

Comment:
suggest to add their config reference in filters.md and controller.md to here...